openshift/s2i/nginx-runtime/nginx.conf.template

```diff
@@ -55,6 +55,10 @@ http {
             proxy_pass          %API_URL%;
         }
 
+        location /ogs-internal {
+            proxy_pass          %GEOSERVER_URL%;
+        }
+
```

Comment:
Putting my comment here because GitHub doesn't let me comment on the relevant line (43)
I believe we need to add "blob"  to the CSP here too right? just like `index.html`?

Which brings me to my next suggestion. I was thinking it would be useful to use an env var for the CSP (like the API_URL, etc). That way we could customize it without having to commit to git and create a new build in OpenShift. Just change the env var in OCP and it would re-apply. What do you think? 

Given time constraints this improvement could come in a future PR....